tasks.py

```diff
@@ -20,20 +32,24 @@ def is_truthy(arg):
     return bool(strtobool(arg))
 
 
-COMPOSE_FILES = ["docker-compose.yml", "../docker-compose.override.yml"]
-if os.getenv("NAUTOBOT_USE_MYSQL"):
-    COMPOSE_FILES.append("docker-compose.mysql.yml")
-
+# Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
+# Variables may be overwritten in invoke.yml or by the environment variables INVOKE_NAUTOBOT_GOLDEN_CONFIG_xxx
 namespace = Collection("nautobot_golden_config")
 namespace.configure(
     {
         "nautobot_golden_config": {
             "nautobot_ver": "1.2.1",
```

Comment:
```suggestion
            "nautobot_ver": "latest",
```

Reply:
Reminder that `latest` for Nautobot docker images points to the latest commit in the `develop` branch - if you want the latest official release, use `stable` instead.

Reply:
@glennmatthews the issue is dev envs are build based on poetry so the docker image is ignored. Latest will actually pull the latest release from PyPI.